examples/Station/Program.cs

```diff
@@ -31,6 +31,14 @@ public static async Task Main(string[] args)
                     });
 
                 Console.WriteLine("Station created successfully...");
+
+                await client.AttachSchema(stationName: station.Name, schemaName: "test-schema-01");
```

Comment:
example files shouldn't include every feature of the SDK so please stick to the examples of the other SDKS

Reply:
Got it. Fixed